interface/super/edit_globals.php

```diff
@@ -797,8 +802,92 @@ function checkBackgroundServices()
 ?>
 
 <script>
+
+//
+function isInRange(input, max){
+    let isValid = false;
+    if(/^\d+$/.test(input)){
+        let num = parseInt(input, 10);
+        if (num >= 0 && num <= max){
+            isValid = true;
+        }
+    }
+    return isValid;
+}
+
+// Determines IP address validity
+function IPvalidation(input_IPs){
+    let IPmax = 255;
+    let CIDRmax = 32
+    let ips = input_IPs.split("\n");
+    for (let j = 0; j < ips.length; j++){
+        let parts = ips[j].split(".");
+        if(parts.length != 4){
+            return false;
+        }
+        if(parts[3].includes("/")){         // contains CIDR notation
+            for(let i = 0; i < 3; i++){
+                if(!(isInRange(parts[i], IPmax))){
+                    return false;
+                }
+            }
+            // checks if the final fourth of the ip is correct. should have the format 255/32
+            let subparts = parts[3].split('/');
+            if(subparts.length != 2){
+                return false;
+            }
+            // IP range is between 0 and 255
+            if(!(isInRange(subparts[0], IPmax))){
+                return false;
+            }
+            // CIDR notation is between 0 and 32
+            if(!(isInRange(subparts[1], CIDRmax))){
+                return false;
+            }
+        }
+        // validates if the ip does not have CIDR notation
+        else{
+            for(let i = 0; i < 4; i++){
+                if(!(isInRange(parts[i], IPmax))){
+                    return false;
+                }
+            }
+        }
+    }
+    return true;
+}
+
 $(function () {
     tabbify();
+
+    let elm = document.getElementById("white_list");
+    let form = document.getElementById('theform');
+    let is_valid = true;
+
+    // Create and style the helper text element
+    let helper_text = document.createElement("span");
```

Comment:
Check out https://getbootstrap.com/docs/4.6/components/forms/#validation as a way to better handle validation styling.